scss/main.scss

```diff
@@ -102,3 +102,5 @@ nav {
     }
   }
 }
+
+@import 'responsive';
```

Comment:
@jehna aha, got it, my bad. in this case, let's do what you're suggesting. let's use this `_responsive.scss` file, but let's move all CSS elements from `main.scss` to something like `_grid.scss`. thus, `main.scss` will contain only `@import` statements.